dask_cuda/__init__.py

```diff
@@ -19,14 +17,6 @@
 from .proxify_device_objects import proxify_decorator, unproxify_decorator
 
 
-if not DASK_EXPR_ENABLED:
-    raise ValueError(
-        "Dask-CUDA no longer supports the legacy Dask DataFrame API. "
-        "Please set the 'dataframe.query-planning' config to `True` "
-        "or None, or downgrade RAPIDS to <=24.12."
-    )
```

Comment:
Moved this logic into `explicit_comms.dataframe.shuffle` since that's really the only place in Dask-CUDA where dask-expr matters.